soap/soap.php

```diff
@@ -271,8 +400,20 @@ public function __construct ($wsdl, array $options = null) {}
 	 * @since 5.0.1
 	 */
 	#[Deprecated]
+	#[PhpStormStubsElementAvailable(to: '5.6')]
 	public function __call ($name, $args) {}
 
+	/**
+	 * @link https://php.net/manual/en/soapclient.call.php
+	 * @param string $name
+	 * @param array $args
+	 * @return mixed
+	 * @since 5.0.1
+	 */
+	#[Deprecated]
+	#[PhpStormStubsElementAvailable(from: '7.0')]
```

Comment:
Let's use `LanguageLevelTypeAware` instead of `PhpStormStubsElementAvailable` to get rid of duplicated entities